enhancements/console/quick-starts.md

```diff
@@ -212,6 +223,27 @@ metadata:
     console.openshift.io/name: explore-serverless
 ```
 
+#### Disabling Quick Starts
+
+1. `operator.openshift.io~v1~Console` CRD provides a `spec.customization` property where an admin will add a property `disabledQuickStarts` with the list of Quick Start Ids to be hidden.
+2. Extend the openshift-console operator so that the console resource data are available in the console ConfigMap and provide them as SERVER_FLAGS.
+3. The Quick Starts listed in `window.SERVER_FLAGS.disabledQuickStarts` will not be visible on the UI
```

Comment:
Remove implementation details that are only relevant to console consuming this data.

```suggestion
`operator.openshift.io/v1` / `Console` CRD provides a `spec.customization` property where an admin will list the Quick Starts to be disabled.
```

Reply:
done